tests/kani/Intrinsics/MaxNum/maxnumf32_fixme.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Check that `maxnumf32` returns the minimum of two values, except in the
+// following cases:
+//  * If one of the arguments is NaN, the other arguments is returned.
+//  * If both arguments are NaN, NaN is returned.
+#![feature(core_intrinsics)]
+
+// Note: All test cases are failing with the following error:
+// ```
+// Check X: fmaxf.assertion.1
+//          - Status: FAILURE
+//          - Description: "Function with missing definition is unreachable"
+//          - Location: <builtin-library-fmaxf> in function fmaxf
+// ```
+// This is because the `fmaxf` definition is not being found, either because
+// Kani does not produce the right expression (which is strange, because it's
+// doing the same for similar expressions and they work) or CBMC is not picking
+// it for some reason.
+// Tracked in https://github.com/model-checking/kani/issues/1025
```

Comment:
Michael seemed to believe the issue there was fixed, have you talked to him about this case?

Reply:
I added a comment in #1025 about this. In principle, all these builtins should be fixed, but `fmaxf` is not working for some reason.

Reply:
`fmaxf` and `fmaxl` were affected, https://github.com/diffblue/cbmc/pull/6904 fixes this.

Reply:
... which is now merged, and will be part of the next release.

Reply:
Since this is being tracked in #1025, I'm going to merge this PR in its current state. This means that the test will work if all goes well when the CBMC version is upgraded, causing the regression to fail unless the test is renamed @zhassan-aws @tedinski

Reply:
Sounds good!